cdap-docs/developers-manual/source/advanced/adapters.rst

```diff
@@ -0,0 +1,140 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+================
+Adapters
+================
+
+Adapters read data from one or more data sources, perform a transformation, then
+write to one or more data sinks. They may be used for ETL purposes, or any situation
+where you want to pipe data from one data source to another.
+
+CDAP currently provides a stream-conversion adapter that regularly reads data from
+a Stream, then writes the data to a TimePartitionedFileSet in avro format. The output
+can then be explored through Hive or Impala, assuming a compatible schema is used. 
+
+Each Adapter type is implemented as a CDAP Application, with each Adapter instance as
```

Comment:
should this be "Adapter-type"? Or can we just dispense with "type", and use "Adapter"?